bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/altair/AltairRule.java

```diff
@@ -15,6 +15,7 @@
 import edu.gemini.shared.util.immutable.Option;
 import edu.gemini.shared.util.immutable.Trio;
 import edu.gemini.shared.util.immutable.Tuple3;
+import edu.gemini.skycalc.Coordinates;
```

Comment:
@cquiroz I think I would have done the same thing as Florian here.  The new target model isn't hooked up yet to the science program model so it would mean converting to/from.  It's probably just as well to do all of that in one go.  In your catalog work, using the new target model makes a lot of sense because it is a major new effort and you're basically replacing the now redundant but less capable `SkyObject`.  Here I think I would probably just wait.